cc_utilities/redcap_sync.py

```diff
@@ -312,8 +310,8 @@ def get_commcare_cases_with_acceptable_interview_dispositions(
     accepted_external_ids = []
     external_ids = df[external_id_col].to_list()
     for ext_id in external_ids:
-        # Get cases in CommCare to compare interview_disposition.
-        # Querying CDMS would be a favorable source of truth for this, but did
+        # Get cases in CommCare to compare interview_disposition. Querying
+        # the SQL mirror would be a favorable source of truth for this, but did
```

Comment:
Oops, I keep forgetting this subtle yet important distinction. Thanks for touching that up! :blush: